modules/two_factor_authentication/app/views/two_factor_authentication/my/two_factor_devices/index.html.erb

```diff
@@ -25,18 +25,26 @@
   </section>
 <% end %>
 
-<% breadcrumb_paths(t(:label_my_account), t('two_factor_authentication.label_two_factor_authentication')) -%>
-<%= toolbar title: t('two_factor_authentication.label_two_factor_authentication') do %>
-  <li class="toolbar-item">
-    <%= link_to new_my_2fa_device_path,
-          { class: 'button -primary',
-            aria: {label: t('two_factor_authentication.label_device')},
-            title: t('two_factor_authentication.devices.add_new')} do %>
-      <%= op_icon('button--icon icon-add') %>
-      <span class="button--text"><%= t('two_factor_authentication.label_device') %></span>
-    <% end %>
-  </li>
-<% end %>
+<%=
```

Comment:
That looks a bit weird with the banner above. I'd rather move it below the SubHeader
<img width="1878" alt="Bildschirmfoto 2024-07-26 um 11 55 37" src="https://github.com/user-attachments/assets/0a8905d1-aa82-4aee-b801-de1367176a6b">